index.js

```diff
@@ -55,8 +45,24 @@ const updateMap = (mapToDisplay) => {
       const vectorLayer = new VectorLayer({
         source: new VectorSource({
           features: (new GeoJSON()).readFeatures(data)
-        }),
-        style: styleFunction
+        })
+      });
+
+      vectorLayer.getSource().forEachFeature(function(feature) {
+        styles = {
+          'Polygon': new Style({
```

Comment:
You're modifying a global variable here. This should either be a local constant, or use a single global constant if it's always the same.